content/about/agreements.md

```diff
@@ -133,7 +133,13 @@ Some links that might be useful to understand best practices for giving and rece
 
 Feedback for jovo can be provided directly in the #jovofeedback slack channel, including anonymous feedback.
 
+#### Contacting People Outside the Lab With Questions
 
+We agree that contacting people outside the lab with questions is effectively asking them to spend their limited time helping us. Since we would prefer to be the ones offering help rather than receiving it, we agree, prior to asking them for help, to:
+  1. Exhaust the resources within ourselves trying to figure out the answer to our problem (this means searching the literature, exploring and writing code, reading books, etc).
```

Comment:
exhaust the resources? seems a bit extreme to me... would that mean i have to spend months trying to learn http before I'm allowed to make an issue asking a question on neuroglancer?

Reply:
less extreme word than "exhaust", maybe? "use our own resources"?

Reply:
better, but to me the important part is that we consider what it would take for us to solve the problem, and consider what we think it would take of them to help us. If a little effort on their part, (e.g. sending us a link to a repo), saves us a ton of time (e.g. trying to build code according to a paper), and we are confident that the task is worth doing (e.g. we're gona spend more than 10 seconds looking at the code before deciding too annoying to use), then I think that's ok.

At the end of the day, it's a judgement call so I don't think you can codify this. But a reminder to consider the costs to us vs them is important.

Reply:
@tathey1 Ok, just addressed this.